.github/workflows/ci.yml

```
@@ -81,7 +81,8 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           directory: empty
-          name: 'Windows (${{ matrix.python }}, ${{ matrix.arch }}${{ matrix.extra_name }})'
+          token: 87cefb17-c44b-4f2f-8b30-1fff5769ce46
```

@A5rocks from the link you provided:

> _A malicious actor would be able to upload incorrect or misleading coverage reports to a specific repository if they have access to your upload token, but would not be able to pull down source code or make any code changes._

So with that in mind, I'd suggest using repository secrets to store an actual token instead of displaying it publicly.

Here, it'd look like
```suggestion
          token: ${{ secrets.CODECOV_TOKEN }}
```
and it'd be sanitized in the logs, addressing said problem.

Though, somebody should probably revoke/reset the token exposed here. I suppose only @njsmith can add repo secrets?